libtrellis/tools/ecppll.cpp

```diff
@@ -422,7 +484,7 @@ void write_pll_config(const pll_params & params, const string &name, ofstream& f
   file << "        .CLKOP_ENABLE(\"ENABLED\"),\n";
   file << "        .CLKOP_DIV(" << params.output_div << "),\n";
   file << "        .CLKOP_CPHASE(" << params.primary_cphase << "),\n";
-  file << "        .CLKOP_FPHASE(0),\n";
+  file << "        .CLKOP_FPHASE(" << params.primary_fphase << "),\n";
```

Comment:
Why was 0 used here? The calcuated fphase is different.

Reply:
Currently this **DOES NOT** work correctly. Please advice @gatecat

Reply:
Thanks, I'll need to take some time to look into this as it's been a good while since I touched the ECP5 PLLs. I will keep this on my list though.